### PR TITLE
# 19 Reformat pbm file

### PR DESCRIPTION
**Pull request by marc4492**
Convert pbm file to csv with right height and width

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+*.pyc
+
+\.idea/inspectionProfiles/Project_Default\.xml
```
Comment (IanLalonde) [inline .gitignore]: unimportant

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+*.pyc
+
+\.idea/inspectionProfiles/Project_Default\.xml
+
+\.idea/encodings\.xml
```
Comment (IanLalonde) [inline .gitignore]: unimportant

**Comment by marc4492** [comment]
connect #19

**Comment by marc4492** [comment]
closes #19

**Pull request merged**